README.md

```diff
@@ -379,7 +379,7 @@ OPTIONS:
        -t <int>        number of threads                                      (default = 48)
        -v              print version number                                   
        -E <e-value>    E-value threshold for homology search                  (default = 1e-5)
-       -S <0 to 1>     AMPlify score threshold for amphibian AMPs             (default = 0.90)
+       -S <3.0103 to 80>     AMPlify score threshold for amphibian AMPs             (default = 10)
```

Comment:
Range was 0 to 1 before - wouldn't the equivalent to this transformed score range be 0.5-1?

Reply:
The AMPlify score of a peptide can be zero, but since it's an **AMP** threshold I put the lower bound as Chenkai's previously set score >= 0.5, And then following the transformation at the time: -10log(1-0.5) = 3.0103, and -10log(1-0.99999999) = 80. Was the transformation equation modified in a recent release of AMPlify?

Reply:
Yup, it was integrated in v1.1.0 (to have the transformed score)

But we had <0 to 1> previously looks like? Should that have been <0.5 to 1> all along?

Reply:
I think so! Doesn't make sense to set a threshold less than 0.5 if that's what's being defined as an AMP. I remember making the changes in the documentation somewhere but I guess they never got pushed

Reply:
Ok great - that makes sense! So we'll just leave it with Lia's corrected range then :)